src/app/components/star-rating/star-rating.less

```diff
@@ -2,9 +2,9 @@ average-star-rating, star-rating {
   position: relative;
   display: block;
 
-  .rating {
+  ul.rating {
     margin: 0;
-    padding: 0;
+    padding: 0!important;
```

Comment:
This is due to a list overriding this property. Not good practice but in that case we really don't want any external css playing with the ratings